showTotalLightningBalance.sh

```diff
@@ -59,76 +58,46 @@ PROFIT_AND_LOSS=$((LN_EARNED_FEES_IN_SATS - ONCHAIN_TX_FEES))
 LN_SPEND=$((LN_PAYMENTS + LN_PAYMENTS_FEES))
 
 ##############################################################################
-# Sats to BTC 
+# Sats to BTC
 ##############################################################################
 
-TOTAL_BALANCE_BTC=$(printf %.3f\\n "$(($TOTAL_BALANCE))e-8")
-LN_LOCAL_BALANCE_BTC=$(printf %.3f\\n "$(($LN_LOCAL_BALANCE))e-8")
-LN_REMOTE_BALANCE_BTC=$(printf %.3f\\n "$(($LN_REMOTE_BALANCE))e-8")
-LN_TOTAL_BALANCE_BTC=$(printf %.3f\\n "$(($LN_TOTAL_BALANCE))e-8")
-LN_INVOICES_BTC=$(printf %.3f\\n "$(($LN_INVOICES))e-8")
-LN_COMMIT_FEES_BTC=$(printf %.3f\\n "$(($LN_COMMIT_FEES))e-8")
-LN_PAYMENTS_BTC=$(printf %.3f\\n "$(($LN_PAYMENTS))e-8")
-LN_PAYMENTS_FEES_BTC=$(printf %.3f\\n "$(($LN_PAYMENTS_FEES))e-8")
+TOTAL_BALANCE_BTC=$(printf %.8f\\n "$((TOTAL_BALANCE))e-8")
+LN_LOCAL_BALANCE_BTC=$(printf %.8f\\n "$((LN_LOCAL_BALANCE))e-8")
+LN_REMOTE_BALANCE_BTC=$(printf %.8f\\n "$((LN_REMOTE_BALANCE))e-8")
+LN_TOTAL_BALANCE_BTC=$(printf %.8f\\n "$((LN_TOTAL_BALANCE))e-8")
 
 ##############################################################################
-# PRINT 
+# PRINT
 ##############################################################################
 
-# turn '0 sats' into '0.000 sats' to keep table nicely formatted
```

Comment:
no longer needed as sats and BTC are aligned in a way thats clear, what is realtionship between them
```
    7000000 sats   
 0.07000000 BTC 
         77 %   
```